QueAns/admin.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib import admin
+from .models import Question,Meetings,Message,Voting,Satisfaction
+
+
+class MeetingsAdmin(admin.ModelAdmin):
+    def has_change_permission(self, request, obj=None):
```

Comment:
Why do we need this method? What if we don't have this?

Reply:
Edited and removed this